addOns/ascanrulesBeta/src/main/java/org/zaproxy/zap/extension/ascanrulesBeta/SessionFixationScanRule.java

```diff
@@ -989,8 +989,7 @@ public void scan() {
                         log.debug(
                                 "The URL parameter [{}] was NOT set in any links in the response, when {()} URL param [{}] was set to NULL in the request, so it is likely not a session id field",
```

Comment:
Not sure, but is it the log message that needs updating in this case?
```suggestion
                                "The URL parameter [{}] was NOT set in any links in the response, when ({}) URL param [{}] was set to NULL in the request, so it is likely not a session id field",
```